tests/components/Microsoft.DotNet.UpgradeAssistant.MSBuild.Tests/TargetFrameworkIdentifierTests.cs

```diff
@@ -80,5 +82,32 @@ public void TfmCompare(string target, string tfm, int expected)
             static TargetFrameworkMoniker? Create(string? input)
                 => input is null ? null : new TargetFrameworkMoniker(input);
         }
+
+        [InlineData("net6.0-windows", "net5.0-windows", "net6.0")]
+        [InlineData("net6.0-windows", "net6.0", "net5.0-windows")]
+        [InlineData("net6.0-windows", "net5.0", "net6.0-windows")]
+        [InlineData("net6.0-windows10.0.5", "net5.0", "net6.0-windows10.0.5")]
```

Comment:
Error	NETSDK1140	10.0.5.0 is not a valid TargetPlatformVersion for Windows. Valid versions include:
10.0.19041.0
10.0.18362.0
10.0.17763.0
8.0
7.0